src/views/DocumentCompare.vue

```diff
@@ -76,6 +77,20 @@ export default {
 			documents: undefined
 		};
 	},
+	computed: {
+		backToPrev: function () {
+			if (prevRoute.name === 'graph') return { path: prevRoute.fullPath };
+			if (prevRoute.name === 'document')
+				return { path: prevRoute.fullPath };
+			return {
+				name: 'matches',
+				params: { result: this.resultId, document: this.documentId }
+			};
+		},
+		documentId: function () {
+			return this.$route.params.document;
```

Comment:
You already have this one in documentAId varaiable. Please unify the approach - either use that one, or simply use route params document on the return...